tests/migrations/test_autodetector.py

```diff
@@ -2395,3 +2395,14 @@ def test_mti_inheritance_model_removal(self):
         self.assertNumberMigrations(changes, 'app', 1)
         self.assertOperationTypes(changes, 'app', 0, ['DeleteModel'])
         self.assertOperationAttributes(changes, 'app', 0, 0, name='Dog')
+
+    def test_delete_unmanaged_model(self):
+        """
+        #29987 - Keep track of the deletion on unmanaged models.
+        """
```

Comment:
We try not to include ticket references in docstrings unless the ticket has a lot of context that the test cannot convey.

Reply:
acknowledged